modules/date-localizer/test/spec.coffee

```diff
@@ -3,79 +3,221 @@ describe 'dateTimeLocalizerMoment', ->
 
 describe 'dateTimeLocalizer', ->
   localizer = hx.dateTimeLocalizer()
-
+  zeroPad = hx.format.zeroPad(2)
   testDate = new Date(1452130200000) # Thu Jan 07 2016 01:30:00 GMT+0000 (GMT)
   zeroHourDate = new Date(1452124800000) # Thu Jan 07 2016 00:00:00 GMT+0000 (GMT)
   invalidDate = new Date('Invalid Date')
+  # 0 - London, -120 - Sofia, -60 Krakow, -540 Tokio, 420 Pinal (Arizona/United states)
+  timezonesOfsets = [0, -120, -60, -540, 420]
+
+  getHexaGonTimeZoneOffset = ->
+    hexagonTimeZoneOffset = hx.preferences.timezone().replace(/^UTC\+?/,'').split(":").map (item, index) ->
+      if index is 0
+        Number(item) * 60
+      else
+        Number(item)
+    .reduce (current, previous) ->
+      current + previous
+    hexagonTimeZoneOffset
```

Comment:
I prefer this method, although if you could pull out the regex (`/(.)(\d{2}):(\d{2})$/`) to a variable at the top (e.g. next to `zeroPad`) it would be slightly more efficient 👍